src/components/HomePage/UpcomingEvents.js

```diff
@@ -1,26 +1,36 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
 import Container from '@material-ui/core/Container';
-import Grid from '@material-ui/core/Grid';
 import Button from '@material-ui/core/Button';
+import Divider from '@material-ui/core/Divider';
 import { Link } from 'gatsby';
 import workshopSchedule from '../../data/WorkshopSchedule';
-import TimeSlot from './TimeSlot';
+import Event from './Event';
+import { currentTimeZoneShort } from '../../utils/timezone_names.js';
+
+const timeFormatter = new Intl.DateTimeFormat('en-US', { hour: 'numeric', minute: '2-digit' });
+
 
 const useStyles = makeStyles(theme => ({
 	title: {
 		fontFamily: theme.typography.fontFamily,
 		fontWeight: 'bold',
-		paddingTop: '24px',
-		paddingBottom: '16px'
+		padding: '1.2em 0'
 	},
 	eventsContainer: {
-		paddingTop: '16px',
-		paddingBottom: '24px'
+		display: 'flex',
+		flexDirection: 'column',
+		justifyContent: 'center'
 	},
 	moreWorkshopsButton: {
-		marginTop: '8px'
+		alignSelf: 'center',
+		marginTop: '3.5em'
+	},
+	divider: {
+		margin: '3.5em 0 3.5em 3em',
```

Comment:
We shouldn't have as much vertical margins on mobile devices. Additionally, I think symmetrical _x_-margins look a bit better… what do you feel? Finally, we should use `theme.spacing()` for these spacing-type properties.

Consider something like:
```
divider: {
  margin: theme.spacing(7),
  [theme.breakpoints.down('sm')]: {
    marginLeft: theme.spacing(3)
  }
```

Reply:
The divider to the end of the container is how it is in the [designs](https://www.figma.com/file/sC4tsO37gn2qgn1ol1yfZq/HOTH-Website?node-id=0%3A1). I guess it's to better separate the descriptions? Or maybe to make things seem more rectangular since there's a lot of white space already? 

I don't have a strong preference, I'll leave it for now and change the implementation to use `theme.spacing()` and also to have mobile have smaller margins. We can decide as a group which one we like better!